lib/lrama/grammar/stdlib.y

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************
+
+  stdlib.y
+
+  This is lrama's standard library. It provides a number of
+  parameterizing rule definitions, such as options and lists,
+  that should be useful in a number of situations.
+
+**********************************************************************/
+
+/*
+ * program: option(number)
+ *
+ * =>
+ *
+ * program: option_number
+ * option_number: ε
+ * option_number: number
+ */
+%rule option(X): /* empty */
+               | X
+               ;
+
+/*
+ * program: list(number)
+ *
+ * =>
+ *
+ * program: list_number
+ * list_number: ε
+ * list_number: number list_number
+ */
+%rule list(X): /* empty */
+             | X list(X)
```

Comment:
The order of RHS tokens are changed, is this intentional?

Reply:
I changed it to be the same as the order of Menhir, but would the original order be better?
https://gitlab.inria.fr/fpottier/menhir/-/blob/20230608/src/standard.mly?ref_type=tags#L209

Reply:
It's interesting. I guess Menhir's rule comes from the fact that new item is appended on the head of list in OCaml, see `(::)` in https://v2.ocaml.org/api/List.html.
On the other hand, new item is appended on the last of the list in Ruby parse.y, for example https://github.com/ruby/ruby/blob/7cf74a2ff28b1b4c26e367d0d67521f7e1fed239/parse.y#L4132.
Therefore I think the original order is better.

Reply:
I see. Thank you for explaining! I changed to use the original order.